core/src/main/java/com/opensymphony/xwork2/security/ExcludedPatternsChecker.java

```diff
@@ -66,18 +66,24 @@
 
         private final boolean excluded;
         private final String excludedPattern;
+        private final boolean log;
```

Comment:
I'm not a fun of such flags as they are quite misleading and control something out of scope of this class.